UPGRADING.md

```diff
@@ -1,7 +1,6 @@
-# Upgrading Tendermint Core
+# Upgrading CometBFT
```

Comment:
This file should be analyzed thoroughly.

It contains multiple references to Tendermint/Tendermint Core, but as old change log entries.

Reply:
I think you should verify with Thane and Sergio, but this should probably be reset to 0.34.27. and the rest left as is.For this PR , I think you can leave it as is, you have it mentioned in the issue tracking what else needs to be done.